charts/stable/uptime-kuma/templates/common.yaml

```
@@ -1 +1,15 @@
-{{ include "tc.common.loader.all" . }}
+{{- include "tc.common.loader.init" . }}
+
+{{- if hasKey .Values "metrics" -}}
+{{- if .Values.metrics.enabled -}}
+{{- $_ := set .Values.podAnnotations "prometheus.io/scrape" "true" -}}
+{{- $_ := set .Values.podAnnotations "prometheus.io/path" "/metrics" -}}
+{{- $_ := set .Values.podAnnotations "prometheus.io/port" (.Values.service.metrics.ports.metrics.targetPort | default 3002 | quote) -}}
```

```suggestion
{{- $_ := set .Values.podAnnotations "prometheus.io/port" (.Values.service.metrics.ports.metrics.targetPort | quote) -}}
```
In this case default won't work if targetPort does not exist (well it could but it gets weird) :)